.github/workflows/workflow-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint GitHub Actions workflow files
+
+on:
+    push:
```

Comment:
Should we also include this for the `wp/X.Y` branches? Going forward, this would lint suggested changes to the workflows in these branches, which are used to push out point releases for older versions of WP when necessary.